split.js

```diff
@@ -241,7 +240,7 @@ var Split = function (ids, options) {
         // Call the drag callback continously. Don't do anything too intensive
         // in this callback.
         if (options.onDrag) {
-            options.onDrag(isVertical, a, b);
+            options.onDrag(direction, a, b);
```

Comment:
You could  do `options.onDrag(direction, elements[this.a].element, elements[this.b].element);` and not define the variables unless they were needed, and save a couple of bytes when minified (if you were that pedantic - I guess I must be!)